test_wazuh/test_fim/test_audit/test_audit_after_initial_scan.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+import pytest
+import os
+import shutil
+
+from wazuh_testing.fim import (LOG_FILE_PATH,
+                               callback_audit_rules_manipulation,
+                               callback_audit_reloaded_rule,
+                               callback_audit_removed_rule,
+                               callback_audit_deleting_rule,
+                               callback_audit_connection_close,
+                               callback_audit_connection)
+from wazuh_testing.tools import (FileMonitor, load_wazuh_configurations,
+                                 check_apply_test, truncate_file)
+
+
+# All tests in this module apply to linux only
+pytestmark = pytest.mark.linux
+
+
+# Variables
+
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
+test_directories = [os.path.join('/', 'testdir1'), os.path.join('/', 'testdir2'), os.path.join('/', 'testdir3')]
+testdir1, testdir2, testdir3 = test_directories
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Configurations
+
+configurations = load_wazuh_configurations(configurations_path, __name__)
+
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Test
+
+@pytest.mark.parametrize('tags_to_apply, folder', [
+    ({'config1'}, '/testdir1')
```

Comment:
Maybe we could try also testdir2 and testdir3

Reply:
Done